### ISSUE TITLE
Once Human

### ISSUE THREAD

**Issue by glenni83** [HansKristian-Work/vkd3d-proton#2235]
There are some major rendering issues in this game. Its been like this for months. It used to play pretty ok on proton-ge 7.55, with a few graphical errors. Then the game got an anti cheat, and we could not use it anymore. It did never work on newer protons. I believe that the proton team has also look at this issue for months now, without a solution.

So some places are rendering pretty good, like the torturial etc. After you get into the open world, every npc, player house, animals starts to stutter heavy. It gets so bad, you get motion sick. Moving and driving is pretty bad in general.

From the look of things, it looks like the whole community has turned upside down on this game, to make it work.

Computer Information:
Manufacturer: ASUSTeK COMPUTER INC.
Model: ROG STRIX X670E-E GAMING WIFI
Form Factor: Desktop
No Touch Input Detected
Processor Information:
CPU Vendor: AuthenticAMD
CPU Brand: AMD Ryzen 7 7800X3D 8-Core Processor
CPU Family: 0x19
CPU Model: 0x61
CPU Stepping: 0x2
CPU Type: 0x0
Speed: 5050 MHz
8 logical processors
8 physical processors
Hyper-threading: Unsupported
FCMOV: Supported
SSE2: Supported
SSE3: Supported
SSSE3: Supported
SSE4a: Supported
SSE41: Supported
SSE42: Supported
AES: Supported
AVX: Supported
AVX2: Supported
AVX512F: Supported
AVX512PF: Unsupported
AVX512ER: Unsupported
AVX512CD: Supported
AVX512VNNI: Supported
SHA: Supported
CMPXCHG16B: Supported
LAHF/SAHF: Supported
PrefetchW: Unsupported
BMI1: Supported
BMI2: Supported
F16C: Supported
FMA: Supported
Operating System Version:
"CachyOS" (64 bit)
Kernel Name: Linux
Kernel Version: 6.12.1-2-cachyos
X Server Vendor: The X.Org Foundation
X Server Release: 12401004
X Window Manager: KWin
Steam Runtime Version:
Client Information:
Version: 1731433018
Browser GPU Acceleration Status: Disabled
Browser Canvas: Unavailable
Browser Canvas out-of-process rasterization: Disabled
Browser Direct Rendering Display Compositor: Disabled
Browser Compositing: Disabled
Browser Multiple Raster Threads: Enabled
Browser OpenGL: Disabled
Browser Rasterization: Disabled
Browser Raw Draw: Disabled
Browser Skia Graphite: Disabled
Browser Video Decode: Disabled
Browser Video Encode: Disabled
Browser Vulkan: Disabled
Browser WebGL: Unavailable
Browser WebGL2: Unavailable
Browser WebGPU: Disabled
Browser WebNN: Disabled
Video Card:
Driver: NVIDIA Corporation NVIDIA GeForce RTX 4090/PCIe/SSE2
Driver Version: 4.6.0 NVIDIA 565.57.01
Desktop Color Depth: 24 bits per pixel
Monitor Refresh Rate: 137 Hz
VendorID: 0x10de
DeviceID: 0x2684
Revision Not Detected
Number of Monitors: 2
Number of Logical Video Cards: 1
Primary Display Resolution: 3840 x 2160
Desktop Resolution: 6400 x 2160
Primary Display Size: 36.22" x 20.35" (41.54" diag), 92.0cm x 51.7cm (105.5cm diag)
Primary VRAM: 24564 MB
Sound card:
Audio device: Nvidia GPU a3 HDMI/DP
Memory:
RAM: 31722 Mb
VR Hardware:
VR Headset: None detected
Miscellaneous:
UI Language: English
LANG: en_GB.UTF-8
Total Hard Disk Space Available: 1181942 MB
Largest Free Hard Disk Block: 941491 MB
Storage:
Number of SSDs: 5
SSD sizes: 4000G,4000G,3840G,250G,0B
Number of HDDs: 0
Number of removable drives: 0

https://filebin.net/owph8spy8wppn5pd

https://www.protondb.com/app/2139460

**Comment by shelterx** [comment]
I read the issue over at Valve, people say the game runs better again?

**Comment by glenni83** [comment]
> I read the issue over at Valve, people say the game runs better again?

No, it runs the same. Its still unplayable. Tested different protons. But no. It might be better for AMD users. But i dont see any change on my nvidia system

**Comment by jcdickinson** [comment]
I am no longer seeing this behavior, though the game does have other issues.

**Comment by K0bin** [comment]
This is not a bug in vkd3d-Proton or DXVK. It's a bug in Wines implementation of d3dcompiler. You can work around it by installing the Windows version of `d3dcompiler_47`. Read up on Winetricks/Protontricks for that.

**Comment by jcdickinson** [comment]
> Please be a little more detailled than "other issues".

Problems with the game itself, including on Windows (from what I can gather from friends). It has so many bugs that it's extremely difficult to tease apart what's exclusive to DXVK. So far as the screenshots in this thread go; those aren't happening anymore.
